src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +13,32 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    public static final String W = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> wordsArray = new ArrayList<>();
+        StringBuilder stringBuilder = new StringBuilder(" ");
+        String line;
+        try {
+            FileReader fileReader = new FileReader(fileName);
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
```

Comment:
where do you close readers? hint: use try with resources

Reply:
done